packages/ui/src/components/Alert/index.tsx

```diff
@@ -15,11 +17,12 @@ const alertStyles = ({
   theme: Theme
   variant: AlertType
 }): SerializedStyles => {
-  const sentiment = theme.colors[variant] || theme.colors.danger
+  const sentiment = theme.colors[variant]
 
   return css`
     background-color: ${sentiment.background};
     color: ${sentiment.text};
+    border-left: 4px solid ${sentiment.borderWeak};
```

Comment:
Should we use the value from the theme instead of hardcoding `4px`?

Reply:
@QuiiBz it's a border not a space, so we can put a hardcoded value. Maybe in the futur we will have border thickness in the theme, until then we do hardcoded 😄